app/io/pathfinder/controllers.scala

```diff
@@ -0,0 +1,98 @@
+package io.pathfinder
+
+import scala.util.{Try,Success,Failure}
+
+import play.api.mvc.Controller
+import play.api.mvc._
+import play.api.libs.json.Json
+
+import scala.collection.JavaConversions.asScalaBuffer
+
+import com.avaje.ebean.{Model,Ebean}
+
+import play.db.ebean.Transactional
+
+import models._
```

Comment:
Please don't use wildcard imports, it makes it harder for other people to figure out what the code is doing.